### DISCUSSION TITLE
Add support for 'unixepoch' function in SQLite

### DISCUSSION THREAD

**Discussion by abaus-vc** [sqlalchemy/sqlalchemy#8354] [Usage Questions]
### Describe the use case

In version 3.38.0, SQLite added the 'unixepoch' datetime conversion function. Trying to use this in a query in the current (1.4.39) release of SQLAlchemy causes an error:

`sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such function: unixepoch`

### Databases / Backends / Drivers targeted

SQLite, versions >= 3.38.0

### Example Use

Create a SQLite database with a table

`CREATE TABLE "test" (
	"id"	INTEGER PRIMARY KEY AUTOINCREMENT,
	"timestamp"	NUMERIC NOT NULL
);`

Add some data to it

`insert into test (timestamp) values (datetime('now'));`

Connect to the db with SQLAlchemy and try to execute the query

`select id, unixepoch(timestamp) from test;`

**Comment by CaselIT** [comment]
Hi,

that error comes from sqlite, so you are probably not using version 3.38.
does running that query work if you run it as `connection.exec_driver_sql('select id, unixepoch(timestamp) from test')`?
